spec/features/kudo_spec.rb

```diff
@@ -21,8 +21,19 @@
     expect(page).to have_content 'Kudo was successfully created.'
     expect(page).to have_content kudo.title
     expect(page).to have_content kudo.content
-    expect(page).to have_content kudo.giver.email
-    expect(page).to have_content kudo.receiver.email
+
+    if kudo.giver.full_name.present?
```

Comment:
People rather avoid `if` conditions in tests - it should be very clear what is expected. Here looks like you expect to have `full_name` on the page, since first_name, last_name are required to be present.

Reply:
When I started this task there were employees who had registered earlier and didn't have full names. Now there are no such users, so it is unnecessary - corrected 🙂